query/aql_compiler.go

```diff
@@ -79,8 +80,8 @@ const (
 
 // Compile returns the compiled AQLQueryContext for data feeding and query
 // execution. Caller should check for AQLQueryContext.Error.
-func (q *AQLQuery) Compile(store memstore.MemStore, returnHLL bool) *AQLQueryContext {
-	qc := &AQLQueryContext{Query: q, ReturnHLLData: returnHLL}
+func (q *AQLQuery) Compile(store memstore.MemStore, returnHLL bool, w http.ResponseWriter) *AQLQueryContext {
```

Comment:
it is quite strange compile needs the http writer as parameter, maybe outside assign to aqlquerycontext?

Reply:
good point, I can change it